app/views/provider_interface/account/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= content_for :title, "Account" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Account</h1>
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">First name</dt>
+        <dd class="govuk-summary-list__value"><%= current_provider_user.first_name %></dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Last name</dt>
+        <dd class="govuk-summary-list__value"><%= current_provider_user.last_name %></dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Email address</dt>
+        <dd class="govuk-summary-list__value"><%= current_provider_user.email_address %></dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Organisations this user has access to</dt>
```

Comment:
Should this be "Organisations you have access to"?

Reply:
It sounds better to me for this page. I will double check with Laura/Adam because I copied it from the prototype https://manage-applications-beta.herokuapp.com/account